### PR TITLE
Update Windows path registration

### PR DESCRIPTION
**Pull request by Spiker985**
Add the Pulsar install directory, and ppm bin directory to the user path - this should hopefully allow downstream steps to be able to shorthand `pulsar` instead of the full path

### CODE REVIEW

**Comment by Spiker985** [comment]
I had to update the test to split the windows test step from the Linux and macOS step. I don't know if that will have an effect on downstream steps. Especially when paired with xvfb.

**Review by confused-Techie — APPROVED** [review]
I'm approving because the changes look solid, and based on our own tests seem to work.

Although prior to merging I think it'd be important to test this PR's changes on an actual package to see what kind of affect it has on testing for Windows.

Then we can use that to determine if this PR includes breaking changes or not, since otherwise I think we should get this one merged no matter what.

Thanks a ton for the contribution, this looks awesome!

**Comment by Spiker985** [comment]
Currently trying this out on a branch for `x-terminal-reloaded`. According to the GitHub Actions docs for [using versioned actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions) you can actually target a branch - which I've done in my [most recent commit](https://github.com/Spiker985/x-terminal-reloaded/commit/96d12224a10aed83fd09faa0690e9bfd5488afdc)

**Comment by Spiker985** [comment]
So, it seems like it was simpler than previously thought to get the Windows PATH fixed for CI - just had to find the right documentation which is definitely easier said than done

**Review by DeeDeeG — COMMENTED** [review]
✅ 

Other than wanting the whitespace-only change nullified on the `test.yml` file, I can register a "lite-approve" on this.

I don't feel qualified to properly approve since I don't understand whether or how the GITHUB_PATH stuff all works. But if you've tested this, I appreciate that. So I am lightly in favor of merging this if it moves this repo forward.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,3 +15,6 @@ jobs:
       - name: Setup Pulsar Editor
         uses: ./
       - run: pulsar -v
+      
+        
+
```
[the remark this answers is not in the document]
Comment (confused-Techie) [reply .github/workflows/test.yml:20]: Gotta say I agree with this point here, but overall amazing work on this PR! Glad to see things are working properly on your tests, and with this last change I'd say lets get this merged and tag a new version

**Comment by Spiker985** [comment]
When I looked, I found [this](https://github.com/electron-userland/electron-builder/issues/6962) which I think is our macOS problem when actually running on CIs such as https://github.com/pulsar-edit/snippets/pull/10 and https://github.com/Spiker985/x-terminal-reloaded/pull/14

**Pull request merged**